p8e-api/src/main/kotlin/io/provenance/engine/grpc/observers/EnvelopeEventObserver.kt

```diff
@@ -190,16 +190,17 @@ class EnvelopeEventObserver(
 
                 affiliateConnection.connectionStatus = CONNECTED
                 affiliateConnection.lastHeartbeat = OffsetDateTime.now()
-            }
```

Comment:
was this transaction extended just to get `affiliateRecord.publicKey.value.toPublicKeyProto()` below?

Reply:
yea, this might be a temp solution for now, until we start work on the clean envelope event observer ticket. (adding auth public key to the signing and encryption key proto).